File: beginners/censor/censor.py
def censor(text, word):
  result = []
  str_list = text.split()
  
  for word_chars in str_list:
    if word_chars == word:
      censor_chars = '*' * len(word)
      result.append(censor_chars)
    else:
      result.append(word_chars)
      
  return ' '.join(result)
  

File: beginners/censor/censor.js
function censor(text, word) {
  const result = [];
  const wordArray = text.split(' ');

  for (let wordItem of wordArray) {
    if (wordItem === word) {
      const censorWord = '*'.repeat(word.length);
      result.push(censorWord);
    } else {
      result.push(wordItem);
    }
  }
  return result.join(' ');
}
